keytool/src/main/scala/org/constellation/keytool/KeyStoreUtils.scala

```diff
@@ -122,4 +153,18 @@ object KeyStoreUtils {
         .use(_.pure[F])
         .attemptT
     } yield keyPair
+
+  def keyPairFromStorePath[F[_]: Sync](
+    path: String,
+    alias: String,
+    storepass: Array[Char],
+    keypass: Array[Char]
+  ): EitherT[F, Throwable, KeyPair] =
+    for {
+      keyPair <- reader(path)
+        .evalMap(unlockKeyStore[F](storepass))
+        .evalMap(unlockKeyPair[F](alias, keypass))
+        .use(_.pure[F])
+        .attemptT
+    } yield keyPair
```

Comment:
Great point, however for the desktop wallet we actually need the ability to pass these args directly since the user will be prompted to provide passwords through the UI. Although `CreateNewTransaction` should have the ability to read these args from env in case that someone wants to use these server side, such as an exchange.